applications/CompressiblePotentialFlowApplication/tests/naca0012_small_adjoint_test/naca0012_small_adjoint_primal_parameters.json

```diff
@@ -0,0 +1,104 @@
+        "solver_type"            : "potential_flow",
+        "domain_size"     : 2,
+        "model_part_name" : "MainModelPart",
+        "model_import_settings"  : {
+            "input_type"     : "mdpa",
+            "input_filename" : "naca0012_small"
+        },
+        "element_replace_settings": {
+                    "element_name":"IncompressiblePotentialFlowElement2D3N",
+                    "condition_name": "PotentialWallCondition2D2N"
+        },
+        "maximum_iterations"     : 10,
+        "echo_level"             : 0,
+        "linear_solver_settings"  : {
+                "solver_type"             : "ExternalSolversApplication.super_lu",
+                "verbosity"               : 0
+        },
+        "volume_model_part_name" : "Parts_Parts_Auto1",
+        "skin_parts"             : ["PotentialWallCondition2D_Far_field_Auto1","Body2D_UpperSurface","Body2D_LowerSurface"],
+        "no_skin_parts"          : []
+    },
+    "processes" : {
+        "initial_conditions_process_list"  : [],
+        "boundary_conditions_process_list" : [{
+            "python_module" : "apply_far_field_process",
+            "kratos_module" : "KratosMultiphysics.CompressiblePotentialFlowApplication",
+            "process_name"  : "FarFieldProcess",
+            "Parameters"    : {
+                "mesh_id"           : 0,
+                "model_part_name"   : "PotentialWallCondition2D_Far_field_Auto1",
+                "inlet_phi"         : 1.0,
+                "velocity_infinity" : [10.0,0.0,0.0]
+            }
+        },{
+            "python_module" : "define_wake_process_2d",
+            "kratos_module" : "KratosMultiphysics.CompressiblePotentialFlowApplication",
+            "process_name"  : "DefineWakeProcess",
+            "Parameters"    : {
+                "mesh_id"         : 0,
+                "model_part_name" : "Wake2D_Wake_Auto1",
+                "upper_surface_model_part_name" : "Body2D_UpperSurface",
+                "lower_surface_model_part_name" : "Body2D_LowerSurface",
+                "wake_direction"       : [1.0,0.0,0.0],
+                "epsilon"         : 1e-9
+            }
+        },{
+            "python_module" : "compute_lift_process",
+            "kratos_module" : "KratosMultiphysics.CompressiblePotentialFlowApplication",
+            "process_name"  : "ComputeLiftProcess3D",
+            "Parameters"    : {
+                "mesh_id"         : 0,
+                "model_part_name" : "MainModelPart",
+                "upper_surface_model_part_name" : "Body2D_UpperSurface",
+                "lower_surface_model_part_name" : "Body2D_LowerSurface",
+                "velocity_infinity" : [10.0,0.0,0.0],
+                "create_output_file": false
+            }
+        },{
+            "python_module" : "placeholder_process",
+            "kratos_module" : "KratosMultiphysics.CompressiblePotentialFlowApplication",
+            "process_name"  : "PlaceholderProcess",
```

Comment:
what is this?

Reply:
It can be removed! I think it was a python script that was created in the beginning of the application and I still have it in the parameters.